package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cqm-execution",
-  "version": "3.1.0",
+  "version": "4.0.0",
```

Comment:
does it overlap with cqm-execution/fhir?

Reply:
not now but it will some point. It is 4.0.1 there. We have not released it yet. I think We should make FHIR one 5.x or 6.x

Reply:
I think the FHIR one being 5.x would work.  It could be interesting if there is a new QDM version post 5.x, but that could still be published as 6.x if need be.